clang/test/Preprocessor/embed_parsing_errors.c

```diff
@@ -94,6 +94,9 @@ char buffer[] = {
 #embed "embed_parsing_errors.c" prefix() // OK: tokens within parens are optional
 #embed "embed_parsing_errors.c" prefix)
 // expected-error@-1 {{expected '('}}
+#embed "embed_parsing_errors.c" prefix()) // expected-error {{expected identifier}}
+#embed "embed_parsing_errors.c" prefix(]) // expected-error {{expected ')'}}
```

Comment:
I wonder if here it would be better to also produce "expected identifier"?

Reply:
That is a good point. We should probably not say expected identifier for the `(()` case. But that is outside of the scope of this pr, it seems. Thanks!